docs/guide/pug-specific-options.md

```diff
@@ -0,0 +1,453 @@
+```
+
+::: warning
+Please note that while this option will process Angular syntax (e.g. `(click)="play()"`), the resulting pug file will throw a syntax error when parsed: `Syntax Error: Assigning to rvalue`
+:::
+
+---
+
+## Closing Bracket Position
+
+`pugClosingBracketPosition`
+
+### Description
+
+Position of closing bracket of attributes.
+
+### Options
+
+**Type:** `choice`  
+**Default:** `'new-line'`
+
+#### `'new-line'`
+
+Closing bracket ends with a new line.
+
+```pug
+input(
+  type="text",
+  value="my_value",
+  name="my_name",
+  alt="my_alt",
+  autocomplete="on"
+)
+```
+
+#### `'last-line'`
+
+Closing bracket remains with last attribute's line.
+
+```pug
+input(
+  type="text",
+  value="my_value",
+  name="my_name",
+  alt="my_alt",
+  autocomplete="on")
+```
+
+---
+
+## Comment Preserve Spaces
+
+`pugCommentPreserveSpaces`
+
+### Description
+
+Change behavior of spaces within comments.
+
+### Options
+
+**Type:** `choice`  
+**Default:** `'keep-all'`
+
+#### `'keep-all'`
+
+Keep all spaces within comments.
+
+```pug
+// ___this _is __a __comment_
+```
+
+#### `'keep-leading'`
+
+Keep leading spaces within comments.
+
+```pug
+// ___this is a comment
+```
+
+#### `'trim-all'`
+
+Trim all spaces within comments.
+
+```pug
+// this is a comment
+```
+
+---
+
+## Sort Attributes
+
+`pugSortAttributes`
+
+### Description
+
+Sort attributes that are not on _beginning_ and _end_ patterns.
+
+### Options
+
+**Type:** `choice`  
+**Default:** `'as-is'`
+
+#### `'as-is'`
+
+Keep the attributes untouched.
+
+```pug
+Foo(a c d b)
+```
+
+#### `'asc'`
+
+Sort attributes ascending.
+
+```pug
+Foo(a b c d)
+```
+
+#### `'desc'`
+
+Sort attributes descending.
+
+```pug
+Foo(d c b a)
+```
+
+---
+
+`pugSortAttributesBeginning`
+
+### Description
+
+Sort attributes by regex patterns to the beginning.
+
+### Options
+
+**Type:** `array`  
+**Default:** `[]`
+
+---
+
+`pugSortAttributesEnd`
+
+### Description
+
+Sort attributes by regex patterns to the end.
+
+### Options
+
+**Type:** `array`  
+**Default:** `[]`
+
+---
+
+[Example](https://github.com/prettier/plugin-pug/issues/22#issuecomment-699509995)
+
+---
+
+## Wrap Attributes
+
+`pugWrapAttributesPattern`
+
+### Description
+
+Define a regex pattern to match attributes against that should always trigger wrapping.
+
+### Options
+
+**Type:** `array`  
+**Default:** `[]`
+
+---
+
+`pugWrapAttributesThreshold`
+
+### Description
+
+Define the maximum amount of attributes that an element can appear with on one line before it gets wrapped.
+
+### Options
+
+**Type:** `choice`  
+**Default:** `-1`
+
+#### `-1`
+
+Only wrap attributes as needed.
+
+```pug
+input(type="text")
+input(type="text", value="my_value", name="my_name")
+```
+
+#### `0`
+
+Always wrap attributes.
+
+```pug
+input(
+  type="text"
+)
+input(
+  type="text",
+  value="my_value",
+  name="my_name"
+)
+```
+
+#### `1`
+
+Allow one unwrapped attribute, wrap two and more.
+
+```pug
+input(type="text")
```

Comment:
Maybe also add an example with two attributes here. Because it is not clear whether the wrapping starts with option=attributes or option=attributes+1

Or maybe change the description a bit:

`Allow up to one attribute to remain in the same line. Wrap all attributes, if there are two or more attributes.`